clearblade.md

```diff
@@ -1186,6 +1187,48 @@ Retrieves the count of items in a collection that match the supplied query
 
 ~~~
 
+# Database
+
+Class: ClearBlade.Database();
```

Comment:
Please maintain this format:
https://github.com/ClearBlade/native-libraries/blob/master/clearblade.md#messagingsubscribetopic-callback

implies: add jsdoc for the query method, callback & also show the structure of the returned data.